webodf/lib/gui/TrivialUndoManager.js

```diff
@@ -195,12 +312,38 @@ gui.TrivialUndoManager = function TrivialUndoManager(defaultRules) {
         eventNotifier.unsubscribe(signal, callback);
     };
 
+
+    /**
+     * @return {!boolean}
+     */
+    this.isDocumentModified = isModified;
+
+    /**
+     * @param {!boolean} modified
+     * @return {undefined}
+     */
+    this.setDocumentModified = function(modified) {
```

Comment:
Do you have a specific usecase in mind for what a user means when they setDocumentModified to true? The logic here is simple enough (setting to true basically means the document will always report modified)... just wondering. If there wasn't a specific use case, the code could be ever-so-slightly simplified to a resetDocumentModified flag instead, that always resets the unmodifiedState to the current state. Possibly easier to document/describe.

But, no major issues with what's here :8ball: